packages/kiota_abstractions/lib/src/serialization/untyped_null.dart

```diff
@@ -0,0 +1,7 @@
+part of '../../kiota_abstractions.dart';
+
+class UntypedNull extends UntypedNode {
+  UntypedNull();
+
+  final dynamic value = null;
```

Comment:
This seems to be unused

Reply:
I changed this code to not having a value at all
and overriding the getValue() method to always return null.